### DISCUSSION TITLE
Add collaborators to a private Qfield cloud instance

### DISCUSSION THREAD

**Discussion by Celpho** [opengisch/QField#3058] [Q&A]
Hello everyone,

I'm setting up a private api, everything works fine except when i add a collaborator to a project, my command seems to work from the ownner point of view but when i login as the collaborator, i can't see the project. Here is my example : 

#1 Check the list of collaborator (owner pov)
curl -X GET "http://localhost:8011/api/v1/collaborators/d850b453-de7d-4899-85ef-c8169c84002c/" -H  "accept: application/json" -H  "Authorization: Token VkNS92bbmWXk3TcGUVLqgw9gNnBZhL6L1KwZ5xdTzKwSL7gOGi6WQlZmNr9coDkfEmLDMo6tVHpj8A3CTTZKqLWfvxieEr8bwnAh"

**[]**

#2 Add the collaborator "test" as "editor" (owner pov)
curl -X POST "http://localhost:8011/api/v1/collaborators/d850b453-de7d-4899-85ef-c8169c84002c/" -H  "accept: application/json" -H  "Content-Type: application/json" -H  "Authorization: Token VkNS92bbmWXk3TcGUVLqgw9gNnBZhL6L1KwZ5xdTzKwSL7gOGi6WQlZmNr9coDkfEmLDMo6tVHpj8A3CTTZKqLWfvxieEr8bwnAh" -d "{ \"collaborator\": \"test\",  \"role\": \"editor\"}"

**{"collaborator":"test","role":"editor"}**

#3 List of projects ("test" pov)
qfieldcloud-cli list-projects
Listing projects…
**User does not have any projects yet.**

Is something wrong with my command? I've tested a similar command with the online api and it worked so what am i missing?

**Comment by suricactus** [comment]
Please make sure you are using the latest master.

Also, the collaborators API is not used internally by QFieldCloud and I am not sure if it still works fine. If you are interested in further investigation, please contact info at opengis.ch

**Reply by Celpho** [reply]
Yes i'm using the latest master (downloaded the 07/07/2022)

I will contect them to see if they know what's causing this. (And i will share the solution)

Thank you!

**Reply by Celpho** [reply]
The latest update has fixed the problem, project appears properly now :)

**Reply by suricactus** [reply]
Great!
